plugins/CoreHome/angularjs/siteselector/siteselector.directive.html

```diff
@@ -29,7 +29,8 @@
                     ng-repeat="site in model.sites"
                     ng-hide="!showSelectedSite && activeSiteId==site.idsite">
                     <a piwik-ignore-click href="{{ getUrlForSiteId(site.idsite) }}"
-                       piwik-autocomplete-matched="view.searchTerm">{{ site.name }}</a>
+                       piwik-autocomplete-matched="view.searchTerm"
+                       ng-bind-html="site.name"></a>
```

Comment:
@tsteur here we are in the loop that shows all the sites. `selectedSiteNameHtml` (which has been removed) wouldn't make sense (unless I miss something?) because `site.name` is the name of the current site in the loop, not the selected one.